src/app/records/page.tsx

```
@@ -5,20 +5,14 @@ import { useRouter } from "next/navigation";
-  type: string;
-  dateIssued: string;
-}
+import { type RecordItem, getRecordTypeName } from "@/types/records";
 
 /**
  * Renders a table row displaying a student's record with actions to view or verify the record.
  *
  * The row is clickable and navigates to the record's detail page. Inline action links allow viewing or verifying the record without triggering row navigation.
  */
-function RecordItem({ id, studentName, type, dateIssued }: RecordItemProps) {
+function RecordItem({ id, studentName, type, dateIssued }: RecordItem) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'RecordItem'. Consider to delete it or rename it.

'RecordItem' is defined here:


(lint/suspicious/noRedeclare)